content/pages/changelog/index.md

```diff
@@ -13,6 +13,19 @@ For historical changelog entries, please see our
 [2019](page://changelog/2019) and
 [2018 release notes](page://changelog/2018).
 
+## 2020-07-20 / Updates to Search Relevance & Performance
+
+In January, updates were released to Box's search [tokenizer](wiki-tokenizer),
+improving the relevancy of search results and performance. This change does not
+effect how metadata is tokenized.
+
+Notable changes to the API-initiated searches include:
```

Comment:
Are these changes API specific?

Reply:
If this is not API specific then I'd rather say.

```suggestion
Notable changes to searches include:
```

Reply:
it isn't API specific. I was just at a loss for how to represent that some of the changes to the webapp search experience wouldn't be reflected in the API

Reply:
My guess is we're not covering the web-app changes here, just the API. So it's kind of self explanatory. Happy to keep as is or just simplify.